content/influxdb/cloud-iox/get-started/setup.md

```diff
@@ -0,0 +1,286 @@
+- `INFLUX_TOKEN`: InfluxDB API token.
+
+```sh
+export INFLUX_HOST=https://cloud2.influxdata.com
+export INFLUX_ORG=<YOUR_INFLUXDB_ORG_NAME>
+export INFLUX_ORG_ID=<YOUR_INFLUXDB_ORG_ID>
+export INFLUX_TOKEN=<YOUR_INFLUXDB_API_TOKEN>
+```
+
+{{% /expand %}}
+
+{{% expand "Command flags" %}}
+
+Use the following `influx` CLI flags to provide required credentials to commands:
+
+- `--host`: [InfluxDB Cloud region URL](/influxdb/cloud-iox/reference/regions/).
+- `-o`, `--org` or `--org-id`: InfluxDB organization name or
+  [ID](/influxdb/cloud-iox/organizations/view-orgs/#view-your-organization-id).
+- `-t`, `--token`: InfluxDB API token.
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+{{% note %}}
+All `influx` CLI examples in this getting started tutorial assume your InfluxDB
+**host**, **organization**, and **token** are provided by either the
+[active `influx` CLI configuration](/influxdb/cloud-iox/reference/cli/influx/#provide-required-authentication-credentials)
+or by environment variables.
+{{% /note %}}
+
+<!------------------------------ END CLI CONTENT ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!----------------------------- BEGIN API CONTENT ----------------------------->
+
+When using the InfluxDB API, provide the required connection credentials in the
+following ways:
+
+- **InfluxDB host**: [InfluxDB Cloud region URL](/influxdb/cloud-iox/reference/regions/)
+- **InfluxDB API Token**: Include an `Authorization` header that uses either 
+  `Bearer` or `Token` scheme and your InfluxDB API token. For example:  
+  `Authorization: Bearer 0xxx0o0XxXxx00Xxxx000xXXxoo0==`.
+- **InfluxDB organization name or ID**: Depending on the API endpoint used, pass
+  this as part of the URL path, query string, or in the request body.
+
+All API examples in this tutorial use **cURL** from a command line.
+To provide all the necessary credentials to the example cURL commands, set
+the following environment variables in your command line session.
+
+```sh
+export INFLUX_HOST=https://cloud2.influxdata.com
+export INFLUX_ORG=<YOUR_INFLUXDB_ORG_NAME>
+export INFLUX_ORG_ID=<YOUR_INFLUXDB_ORG_ID>
+export INFLUX_TOKEN=<YOUR_INFLUXDB_API_TOKEN>
+```
+<!------------------------------ END API CONTENT ------------------------------>
+{{% /tab-content %}}
+    {{< /tabs-wrapper >}} 
+
+6.  {{< req text="(Optional)" color="magenta" >}} **Create a bucket**.
+
+    You can use an existing bucket or create a new one specifically for this
+    getting started tutorial. All examples in this tutorial assume a bucket named
+    **"get-started"**.
+
+    Use the **InfluxDB UI**, **`influx` CLI**, or **InfluxDB API** to create a
+    new bucket.
+
+    {{< tabs-wrapper >}}
+{{% tabs %}}
+[InfluxDB UI](#)
+[influx CLI](#)
+[InfluxDB API](#)
+{{% /tabs %}}
+
+{{% tab-content %}}
+<!------------------------------ BEGIN UI CONTENT ----------------------------->
+
+1.  Visit
```

Comment:
I would use "navigate" or "log in" to the InfluxDB UI at http://localhost:8086 instead of "visit"